chemprop/train/make_predictions.py

```diff
@@ -206,7 +206,8 @@ def predict_and_save(args: PredictArgs, train_args: TrainArgs, test_data: Molecu
 
         # Reshape multiclass to merge task and class dimension, with updated num_tasks
         if args.dataset_type == 'multiclass':
-            preds = preds.reshape((num_tasks))
+            if isinstance(preds, np.ndarray) and preds.ndim > 1:
+                preds = preds.reshape((num_tasks))
             if args.ensemble_variance or args. individual_ensemble_predictions:
```

Comment:
This same error occurs in the next two lines with the ind_preds variable if the --individual_ensemble_predictions argument is used. We should go ahead and indent the next two lines so that they fall under the if statement that you have added.
```suggestion
                if args.ensemble_variance or args. individual_ensemble_predictions:
```